### PR TITLE
[dagit-bb] Add Material icons to storybook example

### PR DESCRIPTION
**Pull request by hellendag**
## Summary

Add Material icons to Storybook, create an `IconWIP` component and `ColorsWIP` map.

## Test Plan

Run storybook, view Icon examples.

### CODE REVIEW

**Comment by hellendag** [comment]
image

File: js_modules/dagit/packages/core/src/ui/Colors.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  Gray400: 'rgba(161, 157, 153, 1)',
+  Gray300: 'rgba(189, 186, 183, 1)',
+  Gray200: 'rgba(218, 216, 214, 1)',
+  Gray100: 'rgba(245, 244, 242, 1)',
+  Gray50: 'rgba(250, 249, 247, 1)',
+  Gray10: 'rgba(35, 31, 27, 0.03)',
+  Blue1: 'rgba(14, 12, 167, 1)',
+  Blue2: 'rgba(79, 67, 221, 1)',
+  Blue3: 'rgba(185, 180, 241, 1)',
+  Blue4: 'rgba(14, 12, 167, 0.08)',
```
Comment (bengotow) [inline js_modules/dagit/packages/core/src/ui/Colors.tsx:18]: Just a stupid nit but it'd be nice to keep these all fully opaque so you can put a Blue4 button inside a Gray900 box (or something like that hypothetically). I think there's a website that we can paste these into and have them all output as opaque colors, so we can try it later if this ever comes up!

File: js_modules/dagit/packages/core/src/ui/Colors.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  Gray700: 'rgba(82, 78, 72, 1)',
+  Gray600: 'rgba(107, 103, 98, 1)',
+  Gray500: 'rgba(134, 131, 127, 1)',
+  Gray400: 'rgba(161, 157, 153, 1)',
+  Gray300: 'rgba(189, 186, 183, 1)',
+  Gray200: 'rgba(218, 216, 214, 1)',
+  Gray100: 'rgba(245, 244, 242, 1)',
+  Gray50: 'rgba(250, 249, 247, 1)',
+  Gray10: 'rgba(35, 31, 27, 0.03)',
+  Blue1: 'rgba(14, 12, 167, 1)',
```
Comment (bengotow) [inline js_modules/dagit/packages/core/src/ui/Colors.tsx:15]: It's slightly confusing that blue1 (smallest number) is the darkest blue but Gray10 (smallest number) is the lightest gray?  I wonder if Gray1 -> Gray90 (being lightest gray) would be more consistent?

**Review by bengotow — APPROVED** [review]
This looks great! I added a few comments but they're all pretty minor nits 🙌

File: js_modules/dagit/packages/core/src/ui/Colors.tsx (hunk 1 above)
Comment (hellendag) [reply js_modules/dagit/packages/core/src/ui/Colors.tsx:18]: The opacity here is by specific request from B&B, but let's keep an eye on it and see if it becomes an issue.

File: js_modules/dagit/packages/core/src/ui/Colors.tsx (hunk 2 above)
Comment (hellendag) [reply js_modules/dagit/packages/core/src/ui/Colors.tsx:15]: Good point, in the specs they're referred to as e.g. `Red 700`, `Red 500`, etc. I can just go with that.

**Pull request merged**